radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java

```diff
@@ -49,14 +50,19 @@
 	private final ImmutableMap<String, String> metaData;
 
 	private Atom() {
-		this.metaData = ImmutableMap.of();
+		this(0);
```

Comment:
Would it be better to use an empty map here, rather than going to the unnecessary trouble of creating a new map with a timestamp that is invalid?

Reply:
Changed as proposed by Laszlo, good catch